contracts/contracts/Franklin.sol

```diff
@@ -730,10 +729,14 @@ contract Franklin {
     function payoutWithdrawNow(address _to, uint16 _tokenId, uint128 _amount) internal {
         if (_tokenId == 0) {
             address payable to = address(uint160(_to));
-            to.transfer(_amount);
-        } else if (_tokenId < governance.totalTokens() + 1) {
+            if (!to.send(_amount)) {
+                balancesToWithdraw[_to][_tokenId] += _amount;
+            }
+        } else if (governance.isValidTokenId(_tokenId)) {
```

Comment:
why not `requireValidTokenId(_tokenId)`? in what situation is a call valid with wrong `_tokenId`?

Reply:
Operator can commit transaction for withdraw with any token id in the range `[0, 2^TOKEN_TREE_DEPTH]` and I am not sure that withdraws for 0 balance are forbidden by circuit.

Reply:
aha, so it's for priority operations?

Reply:
`require` variant is used for submitting priority ops.
`check` is used here, because operator in theory can submit withdraw for invalid token id here.